src/locales/index.ts

```diff
@@ -1,3 +1,3 @@
-export { default } from "./locale";
+export type { default } from "./locale";
```

Comment:
isn't `type` a reserved keyword?

Reply:
The linter is flagging the export and from what I found online you need to use the `type` keyword here to get around it

<img width="717" alt="image" src="https://user-images.githubusercontent.com/44157880/160372862-fbcda143-448d-411d-b20c-9813380fe337.png">